tests/foreman/cli/test_syncplan.py

```diff
@@ -102,17 +103,18 @@ def valid_name_interval_update_tests():
     ]
 
 
-class SyncPlanTestCase(CLITestCase):
+@pytest.mark.usefixtures("setup_org_for_syncplan")
+class TestSyncPlan:
     """Sync Plan CLI tests."""
 
     org = None
+    logger = logging.getLogger('robottelo')
 
-    def setUp(self):
+    @pytest.fixture(scope='class')
+    def setup_org_for_syncplan(self):
```

Comment:
I feel we are stopping step too early here.

It's better for fixture to return object instead of modifying specific class definition. Modifying class is available and often necessary step during unittest -> pytest, but here I don't see the need for it.

`cached=True` parameter seems to put object in cache, so next time same factory function is called, it will return already existing object. With module/class scoped fixture, is there really any need for this?

I suggest to re-use existing `module_org` fixture. Injecting it to all tests with `pytest.mark.usefixtures` is good idea.

Reply:
OK, I will try that, thanks for the review

Reply:
@mirekdlugosz , changes made, please review once more.

I added `module_org` to list of arguments, replacing `self` rather than use `pytest.mark.usefixtures` above every test.

Thank you